pkgs/by-name/js/jspecify/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchMavenArtifact,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "jspecify";
+  version = "1.0.0";
+
+  src = fetchMavenArtifact {
+    groupId = "org.jspecify";
+    artifactId = "jspecify";
+    version = finalAttrs.version;
+    hash = "sha256-H61ua+dVd4Hk0zcp1Jrhzcj92m/kd7sMxozjUer9+6s=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm444 ${finalAttrs.src.jar} $out/share/java/${finalAttrs.pname}-${finalAttrs.version}.jar
+
+    runHook postInstall
+  '';
+
```

```suggestion

  passthru.updateScript = nix-update-script { };

```

Add `nix-update-script`

---

I read about this. I will merge it in and try generating the update script.

---

It should just work for trivial packages.